src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page/workers.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'workers()'
+excerpt: 'Browser module: page.workers method'
+---
+
+This method returns an array of the dedicated WebWorkers associated with the page.
+
```

Comment:
We should probably have a _Returns_ sub heading in here.

Reply:
Resolve in https://github.com/grafana/k6-docs/commit/95398b184f9afe52df89593693d16dfa2d7085c5